PHP-Form-Builder/form/form-display.vm

```diff
@@ -19,9 +19,11 @@
 #end
 #set($validFormFields="#buildValidFormFieldsArray()")
 [system-view:external]
-<?php 
-  define("AUTH_FILE_PATH", "${formConfigContent.getChild('authFilePath').text}");
-  error_reporting(E_ALL);
+<?php
+    #if( $wayToSave.text == "save-to-excel" || $wayToSave.text == "save-to-cms")

```

Comment:
This actually won't work because `$wayToSave` is a loop variable from lines 11-19 above when setting some flags. 

Instead the following will work:

```
#if( $excelSave == "true" || $cmsSave == "true")
```